### ISSUE TITLE
[Bug]: Intermittent permissions error when not connected as user postgres

### ISSUE THREAD

**Issue by FireAndIce68** [timescale/timescaledb#6605]
### What type of bug is this?

Unexpected error

### What subsystems and features are affected?

Query executor

### What happened?

2024-02-05 17:15:27.249 UTC [1124825] app_user@feed ERROR:  permission denied for table compress_hyper_2_132_chunk
2024-02-05 17:15:27.249 UTC [1124825] app_user@feed STATEMENT:  SELECT 
	"Id", ....

The error above occasionally occurs when trying to perform operations as a user which is not the postgres user. I cannot directly reproduce.

### TimescaleDB version affected

2.13.0

### PostgreSQL version used

16.1

### What operating system did you use?

Ubuntu 22.04.3

### What installation method did you use?

Deb/Apt

### What platform did you run on?

On prem/Self-hosted

### Relevant log output and stack trace

```bash
2024-02-05 17:15:27.249 UTC [1124825]  app_user@feed STATEMENT:  SELECT 
	"Id", 
	"TimestampUtc",
	"ReceivedTimestamp",
	"FeedData"
	FROM "Feed"
```

### How can we reproduce the bug?

```bash
The error above occasionally occurs when trying to perform operations as a user which is not the postgres user. I cannot directly reproduce.
```

**Comment by FireAndIce68** [comment]
The only thing I can think of which may be different about these chunks is that they have recently been moved using move_chunk

**Comment by fabriziomello** [comment]
@FireAndIce68 seems this was already fixed in 2.13.1 by this PR: https://github.com/timescale/timescaledb/pull/6439. Can u please update to 2.13.1 and check again?

**Comment by FireAndIce68** [comment]
thank you. Applied the 2.13.1 patch and it seems to working.
